rpc/src/services/base_services.rs

```diff
@@ -239,18 +236,10 @@ pub(crate) fn get_cycle_from_context(block_id: &str, list: ContextList, persiste
     //     }
     // };
 
-    let context = TezedgeContext::new(BlockStorage::new(&persistent_storage), list.clone());
-    let context_index = ContextIndex::new(Some(ctxt_level.try_into()?), None);
-    let context_data = context.get_by_key_prefix(&context_index, &vec!["data/cycle/".to_string()])?;
-
-    // get cylce list from context storage
-    // let cycle_lists: HashMap<String, Bucket<Vec<u8>>> = context_data.clone().into_iter()
-    //     .filter(|(k, _)| k.contains("cycle"))
-    //     .filter(|(_, v)| match v {
-    //         Bucket::Exists(_) => true,
-    //         _ => false
-    //     })
-    //     .collect();
+    let context = TezedgeContext::new(BlockStorage::new(&persistent_storage), persistent_storage.merkle());
```

Comment:
@michal-stanek 
I am just thinking, do we need to make instance of TezedgeContext all the time?
Couldnt we just store reference in RpcServiceEnvironment like persistent_storage is?

Reply:
Everything under TezedgeContext was Arc<> so construction wasn't expensive but I changed it to use a reference anyway so that it looks a bit nicer in base_services.